### DISCUSSION TITLE
建议

### DISCUSSION THREAD

**Discussion by LanFeng11** [dreamncn/Qianji_auto#152] [奇思妙想]
1.建议账单列表出一个未记账的统计，可以全部删除未记账账单
2.建议测试规则时能清晰的知道用了哪个规则，方便定位有问题的规则或冲突重复的规则，比如说我在app数据中测试规则，他会弹出'付款扫码'匹配成功，耗时8ms，或者在日志中体现。账单列表发起记账也一样，可以很好的帮助我们这些半吊子定位问题😂

**Comment by dreamncn** [comment]
建议1∶长按账单列表可以进入未记账统计列表
建议2∶计划中



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年11月6日(周日) 中午12:18
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [Auto-Accounting/Qianji_auto] 建议 (Discussion #152)




 
1.建议账单列表出一个未记账的统计，可以全部删除未记账账单
 2.建议测试规则时能清晰的知道用了哪个规则，方便定位有问题的规则或冲突重复的规则，比如说我在app数据中测试规则，他会弹出'付款扫码'匹配成功，耗时8ms，或者在日志中体现。账单列表发起记账也一样，可以很好的帮助我们这些半吊子定位问题😂
